kotlin-browser/src/main/kotlin/org/w3c/dom/css/sx.kt

```diff
@@ -0,0 +1,13 @@
+package org.w3c.dom.css
+
+import csstype.CssDsl
+import csstype.Properties
+
+inline val ElementCSSInlineStyle.sx: Properties
```

Comment:
Why not `style` instead of `sx`?

Reply:
`sx` is style extension, it means you can use selectors inside, here is just regular style as I can see it